daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Pretty.scala

```diff
@@ -78,14 +62,14 @@ private[lf] object Pretty {
         text("Update failed due to fetch of an inactive contract") & prettyContractId(coid) &
           char('(') + (prettyTypeConName(tid)) + text(").") /
           text(s"The contract had been consumed in sub-transaction #$consumedBy:") +
```

Comment:
You have change quite a bit the type of `consumedBy`, so I think you have change quite a bit the output string.
This is probably not tested :weary:

Reply:
We need test for that, since it used in the scenario service and then in the IDE.